tests/pivots/spreadsheet_pivot/spreadsheet_pivot_side_panel.test.ts

```diff
@@ -469,17 +475,17 @@ describe("Spreadsheet pivot side panel", () => {
 
     await click(fixture.querySelector(".o-pivot-measure .add-dimension")!);
     await click(fixture.querySelectorAll(".o-autocomplete-value")[1]);
-    // don't notify when dynamic pivot is visible
+    // don't notify when only dynamic pivot is visible
     expect(mockNotify).toHaveBeenCalledTimes(0);
 
     // scroll beyond the =PIVOT formula
     setViewportOffset(model, 0, 1000);
-    await click(fixture.querySelector(".o-pivot-measure .add-dimension")!);
-    await click(fixture.querySelectorAll(".o-autocomplete-value")[1]);
-    expect(mockNotify).toHaveBeenCalledTimes(0);
-
+    // check that cell A50 is visible in viewport
+    expect(model.getters.getVisibleCellPositions().map((p) => toXC(p.col, p.row))).toContain("A50");
     // add a static pivot in the viewport
-    setCellContent(model, "A50", "=PIVOT.VALUE(1)");
+    selectCell(model, "A50");
```

Comment:
I modified the test a bit to use an approach that does not require any position check.
 ```typescript
// finds the bottom right visible cell of the viewport
const { bottom: row, right: col } = model.getters.getActiveMainViewport();
setCellContent(model, toXC(col, row), "=PIVOT.VALUE(1)")
```